packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade TypeScript to v5.0 and set `moduleResolution` option to `Node16` ([#3645](https://github.com/MetaMask/core/pull/3645))
+- Bump `@metamask/approval-controller` from `^7.0.0` to `^7.0.2` ([#4517](https://github.com/MetaMask/core/pull/4517), [#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/base-controller` from `^6.0.0` to `^6.0.2` ([#4517](https://github.com/MetaMask/core/pull/4517), [#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/controller-utils` from `^11.0.0` to `^11.0.2` ([#4517](https://github.com/MetaMask/core/pull/4517), [#4544](https://github.com/MetaMask/core/pull/4544))
+- Bump `@metamask/keyring-controller` from `^17.1.0` `to ^17.1.2` ([#4527](https://github.com/MetaMask/core/pull/4527), [#4548](https://github.com/MetaMask/core/pull/4548))
+- Bump `@metamask/rpc-errors` from `^6.2.1` to `^6.3.1` ([#4516](https://github.com/MetaMask/core/pull/4516))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#4516](https://github.com/MetaMask/core/pull/4516), [#4529](https://github.com/MetaMask/core/pull/4529))
```

Comment:
Do we want to include these as well?

``` suggestion
- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#4516](https://github.com/MetaMask/core/pull/4516), [#4529](https://github.com/MetaMask/core/pull/4529))
- Bump `@metamask/accounts-controller` from `^17.2.0` to `^17.2.1` ([#4548](https://github.com/MetaMask/core/pull/4548))
- Bump `@metamask/network-controller` from `^20.0.0` to `^20.1.0` ([#4548](https://github.com/MetaMask/core/pull/4548))

Reply:
Hmm, these shouldn't be dependencies in the first place. They're peer dependencies.

Reply:
Fixed here: https://github.com/MetaMask/core/pull/4556

Landing this would simplify the release process in that we'd be able to omit minor/patch changes to these packages.